openapi/openapi.yaml

```
@@ -62,10 +62,9 @@ components:
 
     PublishableApiKey:
       description: |-
-        Only for the Tokens resource.
+        This authentication method is only applicable for the tokens resource.
```

```suggestion
        This authentication method is applicable to specific operations.
```

There are now about 20 API operations that use this method.